src/v0/util/index.js

```diff
@@ -1279,6 +1279,31 @@ function getFullName(message) {
   return fullName;
 }
 
```

Comment:
you can add quick unit test for this in index.test.js

Reply:
added.